Cargo.toml

```
@@ -49,7 +49,7 @@ strum_macros = ">= 0.16, <= 0.18"
 snafu = ">= 0.5, <= 0.6"
 getset = "0.0.9"
 enum-map = "0.6"
-triple_accel = "0.3.2"
+triple_accel = "0.3.4"
```

Looks good to me! Alternatively, just leave out the patch part?
```suggestion
triple_accel = "0.3"
```